nemo/collections/asr/parts/submodules/causal_convs.py

```diff
@@ -140,23 +131,16 @@ def __init__(
 
     def update_cache(self, x, cache=None, cache_next=None):
         if cache is None:
-            x = F.pad(x, pad=(self._left_padding, self._right_padding))
+            new_x = F.pad(x, pad=(self._left_padding, self._right_padding))
```

Comment:
Let's add "cache is not None and cache_next is not None" to the if block or separate cache and cache_next like before. Both ways are OK.